ang/civicase/activity/services/activity-feed-measurements.service.js

```diff
@@ -17,5 +17,39 @@
 
       return topOffset;
     }
+
+    /**
+     * Returns the collective height of all contact tab children
+     *
+     * @return {Number}
+     */
+    function civicrmContactTabHeight () {
+      var height = 0;
+      var $civicrmContactTabChildren = $('.crm-contact-tabs-list').children();
+
+      _.each($civicrmContactTabChildren, function (child) {
+        height += $(child).height();
+      });
+
+      return height;
+    }
+
+    /**
+     * Set height of the sent element for scrolling
+     *
+     * @param {Object} $elm
+     * @return {Number}
+     */
+    function setHeightOf ($elm) {
```

Comment:
`setHeightOf`, I'm not sure about this name. Can it be a bit more descriptive? I don't understand the function in general so I can't give a suggestion.

Reply:
Also, please use `$element` instead of `$elm`